src/azul/service/manifest_service.py

```diff
@@ -871,9 +876,13 @@ def source_filter(self) -> SourceFilters:
 
     def write_to(self, output: IO[str]) -> Optional[str]:
         sources = list(self.manifest_config.keys())
-        ordered_column_names = [field_name
-                                for source in sources
-                                for field_name in self.manifest_config[source]]
+        ordered_column_names = []
+        for source in sources:
+            field_names = self.manifest_config[source]
+            ordered_column_names.extend(field_names)
+            if source == 'contents.files':
```

Comment:
This feels a little odd. If specific customization of the manfest config are needed for a particular manifest format, the corresponding manifest generator class should override manifest_config to add that.

Then again, it would be OK to default to including it in all formats, so the column should be configured in the `ServiceConfig` instance defined in the plugin. If there is a column naming conflict with an existing BDBag column we have the option to suppress/rename the column there, but this time in the BDBag generator's `manifest_config` override.